bpython/sample-config

```diff
@@ -36,6 +36,10 @@
 # color_scheme = default
 
 # External editor to use for editing the current line, block, or full history
+# Options: vi (vim)
```

Comment:
I'd call that examples. Options would suggest that these are the only supported values.